### ISSUE TITLE
Support for multiple GNSS

### ISSUE THREAD

**Issue by rzinke** [insarlab/MintPy#1169]
**Description of the desired feature**

Currently, MintPy only supports UNR solutions. It would be helpful for it to support multiple (or generic) solutions.

**Describe the solution you'd like**
One flag to select among different supported solutions (e.g., 'UNR', 'ESESES', 'JPL'). And a generic file structure for less common velocity/displacement formats.

**Additional context**
Especially useful for NISAR cal/val.

**Are you willing to help implement and maintain this feature?**

- [X] Yes
- [ ] No

I (rzinke) will provide an initial pull request within 10 days of 27 March 2024.

**Comment by rzinke** [comment]
Hi @yunjunz, thanks for information--this is very helpful, and matches with what I deduced from the code. I'm definitely happy to hear thoughts and work with fellow developers on this.

**Comment by rzinke** [comment]
Hello @yunjunz, @falkamelung, I have finalized [pull request #1173](https://github.com/insarlab/MintPy/pull/1173) in which I have adapted mintpy/objects/gps to work with multiple GNSS data sources. I have tested the objects and functions in the gps module, as well as "downstream" routines such as `view.py`, `tsview.py`, `plot.py`, and `insar_vs_gps.py`. The pull request is waiting for at least one reviewer. What is the process for finding reviewers?
